### ISSUE TITLE
Need possibility to define aria-label for CvTabs contained nav element

### ISSUE THREAD

**Issue by KJaspers** [carbon-design-system/carbon-components-vue#409]
In our app we already have a nav element and our accessibility checker tool reports the following violation: "Multiple 'navigation' landmarks must have unique labels specified with aria-label or aria-labelledby."

Maybe it would be a solution to provide a way to define an aria-label for the nav element in the CvTabs component...

**Comment by lee-chase** [comment]
Assigning $attrs to the nav tab as it seems the most appropriate location.

Will publish later today.

**Comment by KJaspers** [comment]
@lee-chase Works perfectly well, thank you!
